test/e2e/backup-restore/backup_restore_test.go

```diff
@@ -43,7 +44,8 @@ func TestBackupRestoreTransformations(t *testing.T) {
 	defer testKube.CleanNamespaceAndLogOnPanic(t, tutils.Namespace)
 
 	// Create a resource without passing any config
-	clusterName := strcase.ToKebab(t.Name())
+	croppedTestName := regexp.MustCompile(".*/").ReplaceAllString(t.Name(), "")
```

Comment:
Also, no need to crop the test name of `TestBackupRestoreTransformations` as it's not used in the suite.

Reply:
> Instead of repeating this for every test, I think we should add a reusable function to tutils, e.g. CroppedTestName(t *testing.T) string or even just TestName(...).

It struck me too when writing the code but I was feeling lazy, updating...

> Also, no need to crop the test name of TestBackupRestoreTransformations as it's not used in the suite.

True but maybe it would be best to update all the tests to use the function. It'll make it easier to add them in the suites later and at least it'll be consistent.

Reply:
> True but maybe it would be best to update all the tests to use the function. It'll make it easier to add them in the suites later and at least it'll be consistent.

+1 that would be great.